src/main/java/stream/flarebot/flarebot/commands/secret/AnnounceCommand.java

```diff
@@ -0,0 +1,51 @@
+package stream.flarebot.flarebot.commands.secret;
+
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import net.dv8tion.jda.core.entities.Role;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+
+public class AnnounceCommand implements Command {
+
+    @Override
+    public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Message msg, String[] args, Member member) {
+        if(guild.getGuildId().equals("226785954537406464")) {
+            Role r = guild.getGuild().getRoleById(320304080926801922L);
+            r.getManager().setMentionable(true).complete();
+            String message = msg.getRawContent();
```

Comment:
For consistency sake it would be better using FlareBot.getMessage(args, 0);

Reply:
I've had issues with newlines not persisting properly when doing that